bundles/AdminBundle/Resources/public/js/pimcore/document/editables/renderlet.js

```diff
@@ -37,7 +37,7 @@ pimcore.document.editables.renderlet = Class.create(pimcore.document.editable, {
             this.defaultHeight = this.config.defaultHeight;
         }
         if (!this.config.height) {
-            this.config.height = this.defaultHeight;
+            this.config.height = 'auto';
```

Comment:
```suggestion
            this.config.height = this.defaultHeight;
```
This is not necessary, as we're setting the body to auto anyway if a content is set. 
Happens here: https://github.com/pimcore/pimcore/blob/ad99f9a2061365848d15534266310a1b2ed83834/bundles/AdminBundle/Resources/public/js/pimcore/document/editables/renderlet.js#L215